workshop2/xdp_prog.c

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2017 GustavoKatel
+// Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
can you add the source link?

did you just copy the code or did you change it?

Reply:
Added a link for "based on". Now it's more different after the recent changes